WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -133,6 +140,9 @@ public void onErrorResponse(VolleyError error) {
     @Override
     protected void fetchRemoteData() {
         fetchCategories();
+        if (mSite.isJetpackConnected()) {
+            fetchJetpackSettings();
```

Comment:
Doesn't this kind create a race condition? Both fetching settings and fetching jetpack settings happen at the same time and as far as I can tell they change the same model. I don't know if using `synchronized` methods would help, but I think we could have some issues in it's current state.